definitions/npm/semver_v7.x.x/flow_v0.104.x-/semver_v7.x.x.js

```diff
@@ -0,0 +1,836 @@
+// Generated from @types/semver using github.com/aminya/flowgen-package
+declare module "semver" {
+  import type * as semverParse from "semver/functions/parse"
```

Comment:
That's not supported by Flow, check the CI tests.

Reply:
What's the flow-typed compatible replacement?

Reply:
> What's the flow-typed compatible replacement?

I can't fix this if Flow doesn't support an alternative for this syntax.

Reply:
Consider examples like the following for imports of types and vars
import types: https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/redux-thunk_v2.x.x/flow_v0.83.x-/redux-thunk_v2.x.x.js#L23
import vars: https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/webpack_v5.x.x/flow_v0.104.x-/webpack_v5.x.x.js#L3

I believe the overall issue here is that you cannot import all named exports into a flow module, you need to specify all individual vars, and I think you didn't mean to `import type` because looking at your exports you're trying to export vars. Looking at what `semver/functions/parse` does I think you want something like

```
import typeof semverParse from 'semver/functions/parse';
```